src/common/xBuild.ts

```diff
@@ -6,7 +6,7 @@ type XmlElement = {
   [tag: string]: any;
 };
 
-const nsMap = mapKeys(namespaces, (v, k) => {
+const nsMap = () => mapKeys(namespaces, (v, k) => {
```

Comment:
Please make this a regular function, rather than an arrow function. Convention of this library only uses arrow functions for anonymous callback functions.